### ISSUE TITLE
RESULTS

### ISSUE THREAD

**Issue by MZ82020** [shuaishiliu/SGCN#12]
Hello @shuaishiliu, 

When I tested the same code with your pretrained model (checkpoints), I hadn't the same results of the paper. Can you explain me why please? 

Another question, how can I convert the matrix to cordinates and visualize the trajectories in figure 4 and 5 in the paper please? can you share with me the code for the visualization of the trajectories please? 

Thanks

**Comment by shuaishiliu** [comment]
> 
Thanks for your comments. The pretrained models are trained after the paper accpeted. It is our mistake that we do not save the original code version. In addition, this code is implemented based on the [Social-STGCNN](https://github.com/abduallahmohamed/Social-STGCNN) and thus we also overlook to set random seed. So the reproduced models exist performance deviation compared with the CVPR version.   We empirically find the experimental environemnt also will effect the performance such as the GPU version.

Answer to another question: The figure 4 is plotted by model the distribution of sampled 20 trajectories. To get it, you  can use ''matplotlib'' and ''seaborn'' to plot the kde density.   
As for convert trajectory into the pixel, first, you should understand the homography matrix H provided in the offical dataset.   The pixel coordinate multiplies the H and you can get the world coordinate. Otherwise, the pixel coordiante is obtained by multipling the inverse H according to the matrix multiplication. Due to the time limitation of my work, I can not provide the code. If you are interested, we will released the visualized tools when we have finished the extending version of this paper.

**Comment by MZ82020** [comment]
@shuaishiliu Hello, thank you for your response. 

When I would train my own model, can I use the dataset existing in the file in github? It is the same dataset on which you have trained your model??
I don't inderstand what refer the cordinates in your dataset file? Can you explain me please? 

Exemple: 
![image](https://user-images.githubusercontent.com/94976539/143612452-490c0a69-9439-4a98-815a-21112ed8704f.png)

**Comment by Xiejc97** [comment]
@shuaishiliu 
Hi,

As you say, the pixel coordiante is obtained by multipling the inverse H according to the matrix multiplication. After I convert the world coordinates into pixel coordinates, how can I find the corresponding scene picture at this time?

**Comment by KiddoLow** [comment]
@shuaishiliu
Hi, I can not find the position encoding tensor which is added to Etmp in the model. 
Can you explain me please?

**Comment by dengjon** [comment]
Hi, why I can't find the source code file? Did you delete those files?

**Comment by KiddoLow** [comment]
No,I did not delete anything.I still want you to answer my question.
----- 原始邮件 -----
发件人：Pluto ***@***.***>
收件人：shuaishiliu/SGCN ***@***.***>
抄送人：lzc ***@***.***>,  Comment ***@***.***>
主题：Re: [shuaishiliu/SGCN] RESULTS (Issue #12)
日期：2022年02月04日 15点19分


Hi, why I can't find the source code file? Did you delete those files?

&mdash;
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Jayoprell** [comment]
There's no source code in this repo.
